gen-client.sh

```diff
@@ -21,6 +21,10 @@ print(Version(sys.stdin.read()))
 
 generator_version ()
 {
+  if [[ -n "${USE_GENERATOR_VERSION:-}" ]]; then
+    echo "v${USE_GENERATOR_VERSION}"
+    return
+  fi
```

Comment:
I cannot say how big of a disclaimer this feature needs.

Reply:
I've dropped that for now.
I couldn't convince myself that this can be useful at all.